ledger-core/v2/conveyor/conveyor.go

```diff
@@ -335,8 +345,8 @@ func (p *PulseConveyor) PreparePulseChange(out PreparePulseChangeChannel) error
 		if !ctx.CallDirectBargeIn(p.presentMachine.SlotLink().GetAnyStepLink(), func(ctx smachine.BargeInContext) smachine.StateUpdate {
 			return p.presentMachine.preparePulseChange(ctx, out)
 		}) {
-			//p.pdm.unsetPreparingPulse()
-			//close(out)
+			// p.pdm.unsetPreparingPulse()
+			// close(out)
```

Comment:
changes in commented out code :/ just drop it or don't change

Reply:
converted into TODO for further work